calico_rkt/calico_rkt.py

```diff
@@ -1,198 +1,212 @@
-#!/usr/bin/env python
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 from __future__ import print_function
 import socket
 import functools
+import logging
 import json
 import os
 import sys
-from subprocess import check_output, CalledProcessError, check_call
 
+from subprocess import check_output, CalledProcessError, check_call
 from netaddr import IPAddress, IPNetwork, AddrFormatError
+
 from pycalico import netns
 from pycalico.ipam import IPAMClient, SequentialAssignment
 from pycalico.netns import Namespace
-from pycalico.datastore_datatypes import Rules
-from pycalico.datastore import IF_PREFIX, DatastoreClient
+from pycalico.datastore_datatypes import Rules, IPPool
+from pycalico.datastore import IF_PREFIX
 from pycalico.datastore_errors import PoolNotFound
 
-print_stderr = functools.partial(print, file=sys.stderr)
+ETCD_AUTHORITY_ENV = 'ETCD_AUTHORITY'
+LOG_DIR = '/var/log/calico/calico-rkt'
 
+ORCHESTRATOR_ID = "rkt"
+HOSTNAME = socket.gethostname()
+NETNS_ROOT = '/var/lib/rkt/pods/run'
 
-input_ = ''.join(sys.stdin.readlines()).replace('\n', '')
-input_json = json.loads(input_)
+_log = logging.getLogger(__name__)
+_datastore_client = IPAMClient()
```

Comment:
Can you replace this to just be `datastore_client`? Its not really a private value at this point...